sdks/wasm/sdk/Mono.WebAssembly.Build/WasmLinkAssemblies.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could we use existing ILLink tasks https://dotnet.myget.org/feed/dotnet-core/package/nuget/ILLink.Tasks instead of building a new one?

(it could be follow up PR)

Reply:
I am thinking Makayla created a new one because of the special linking of the WebAssembly Bindings and Http.

Reply:
Right. This is just temporary anyways, was going to replace it with a custom task that lets us wire in wasm-specific steps.